components/slideshow.tsx

```diff
@@ -0,0 +1,97 @@
+'use client';
+import Image from 'next/image';
+import { useCallback, useEffect, useState } from 'react';
+import { FaChevronLeft, FaChevronRight } from 'react-icons/fa';
+
+export default function Carousel({ images }: { images: string[] }) {
```

Comment:
Rename to slideshow for consistency.
```suggestion
export default function Slideshow({ images }: { images: string[] }) {
```